configuration/postgres/metadata.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package postgres
+
+import "time"
+
+type metadata struct {
+	maxIdleTime      time.Duration
```

Comment:
Would recommend a more descriptive name. For example "timeout" or something like that

Reply:
changed to maxIdleTimeout